tests/lifecycle/helper/helper.go

```diff
@@ -186,3 +186,24 @@ func DeletePV(persistentVolume string, timeout time.Duration) error {
 
 	return nil
 }
+
+func AppendAnnotationsToPod(pod *corev1.Pod, annotation string, annotationValue string) {
+	m := make(map[string]string)
+
+	for k, v := range pod.Annotations {
+		m[k] = v
+	}
+
+	pod.Annotations = make(map[string]string)
+	pod.Annotations[annotation] = annotationValue
+}
+
+func DeleteRunTimeClasses() error {
+	err := globalhelper.APIClient.RuntimeClasses().DeleteCollection(context.Background(),
```

Comment:
This will remove all the runtimeClasses in the cluster... do you really want to do that?

Reply:
Fixed to delete specific one, thanks